ci-automation/vendor-testing/aws.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+# Copyright (c) 2022 The Flatcar Maintainers.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -euo pipefail
+
+set -x
+
+# Test execution script for the AWS vendor image.
+# This script is supposed to run in the mantle container.
+
+source ci-automation/vendor_test.sh
+
+board="${CIA_ARCH}-usr"
+image_name="ci-${CIA_VERNUM//+/-}"
+aws_instance_type_var="AWS_${CIA_ARCH}_INSTANCE_TYPE"
+aws_instance_type="${!aws_instance_type_var}"
+more_aws_instance_types_var="AWS_${CIA_ARCH}_MORE_INSTANCE_TYPES"
+set -o noglob # there shouldn't be any instance types with asterisks
+              # in it, but…
+more_aws_instance_types=( ${!more_aws_instance_types_var} )
+set +o noglob
+
+vmdk='flatcar_production_ami_vmdk_image.vmdk'
+
+if [[ "${AWS_AMI_ID}" == "" ]]; then
+    if [[ -f "${vmdk}" ]]; then
+        echo "++++ ${CIA_TESTSCRIPT}: using existing ${vmdk} for ${CIA_VERNUM} (${CIA_ARCH}) ++++"
+    else
+        echo "++++ ${CIA_TESTSCRIPT}: downloading ${vmdk}.bz2 for ${CIA_VERNUM} (${CIA_ARCH}) ++++"
+        copy_from_buildcache "images/${CIA_ARCH}/${CIA_VERNUM}/${vmdk}.bz2" .
+        lbunzip2 "${vmdk}.bz2"
+    fi
+
+    AWS_BUCKET="flatcar-kola-ami-import-${AWS_REGION}"
+    trap 'ore -d aws delete --region="${AWS_REGION}" --name="${image_name}" --ami-name="${image_name}" --file="${vmdk}" --bucket "s3://${AWS_BUCKET}/${board}/"' EXIT
+    ore aws initialize --region="${AWS_REGION}" --bucket "${AWS_BUCKET}"
+    AWS_AMI_ID=$(ore aws upload --force --region="${AWS_REGION}" --name="${image_name}" --ami-name="${image_name}" --ami-description="Flatcar Test ${image_name}" --file="${vmdk}" --bucket  "s3://${AWS_BUCKET}/${board}/" | jq -r .HVM)
+    echo "++++ ${CIA_TESTSCRIPT}: created new AMI ${AWS_AMI_ID} (will be removed after testing) ++++"
+fi
+
+run_kola_tests() {
+    local instance_type="${1}"; shift
+    local instance_tapfile="${1}"; shift
+
+    timeout --signal=SIGQUIT 6h \
```

Comment:
```suggestion
    set -x
    timeout --signal=SIGQUIT 6h \
```
I like to see the printed kola command line to be able to run it locally.